packages/opentelemetry-exporter-collector/README.md

```diff
@@ -6,7 +6,7 @@
 [![devDependencies][devDependencies-image]][devDependencies-url]
 [![Apache License][license-image]][license-image]
 
-This module provides exporter for web and node to be used with [opentelemetry-collector][opentelemetry-collector-url].
+This module provides exporter for web and node to be used with [opentelemetry-collector][opentelemetry-collector-url] ver. **0.6.0**
```

Comment:
```suggestion
This module provides exporter for web and node to be used with [opentelemetry-collector][opentelemetry-collector-url] ver. >= **0.6.0**
```

Reply:
@naseemkullah hmm I won't be so sure if this is going to work fine with higher versions so I would rather have fixed version only. I had already some troubles with different versions and docker containers and different configurations between versions. Hard to say if it is safe to say ">=", @dyladan what's your take on this ?

Reply:
I would say, "last tested with version x.y.z". Until we go to 1.0, it is hard to make stability guarantees

Reply:
done